x-pack/plugins/maps/public/classes/styles/vector/components/symbol/icon_stops.test.js

```diff
@@ -4,17 +4,41 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+jest.mock('./icon_select', () => ({
+  IconSelect: () => {
+    return <div>mockIconSelect</div>;
+  },
+}));
+
+jest.mock('../../symbol_utils', () => {
+  return {
+    SYMBOL_OPTIONS: [{ value: 'icon1' }, { value: 'icon2' }],
+    PREFERRED_ICONS: [
+      'circle',
+      'marker',
+      'square',
+      'star',
+      'triangle',
+      'hospital',
+      'circle-stroked',
+      'marker-stroked',
+      'square-stroked',
+      'star-stroked',
+      'triangle-stroked',
+    ],
+  };
+});
+
+import React from 'react';
 import { getFirstUnusedSymbol } from './icon_stops';
```

Comment:
We should prob move these imports up above the mocks. A little awkward importing `React` here when we're only using it on line 9.